teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization)
```

Comment:
The `initialize` method in the `Teacher` class also takes age, name = "Unknown", parent_permission = true as parameters. Kindly add these accordingly.